src/gilded_rose.spec.js

```diff
@@ -31,7 +31,7 @@ describe.only('updating of aged brie', () => {
 
   it('increments in quality by 1', () => {
     const agedBrieArray = updateQuality([agedBrie]);
-    console.log(agedBrieArray[0].quality)
+    console.log(agedBrieArray[0].quality);
```

Comment:
Heads up that the commit type for this is _probably_ `style` instead of `chore` since it looks like the topic of your commits are about code style. Chores are usually for updating NPM packages, installing them, etc. From the [angular contributing docs](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines) (that [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification) are based on).

>style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)